WooCommerce/Classes/ViewRelated/Notifications/Cells/NoteDetailsCommentTableViewCell.xib

```diff
@@ -145,8 +143,8 @@
                 </subviews>
                 <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                 <constraints>
-                    <constraint firstItem="QbZ-zr-lT5" firstAttribute="top" secondItem="p0w-Eh-NgH" secondAttribute="topMargin" id="Rl1-g8-YkH"/>
-                    <constraint firstAttribute="bottomMargin" secondItem="QbZ-zr-lT5" secondAttribute="bottom" id="Vfz-Zj-1v5"/>
+                    <constraint firstItem="QbZ-zr-lT5" firstAttribute="top" secondItem="p0w-Eh-NgH" secondAttribute="topMargin" constant="4" id="Rl1-g8-YkH"/>
+                    <constraint firstAttribute="bottomMargin" secondItem="QbZ-zr-lT5" secondAttribute="bottom" constant="6" id="Vfz-Zj-1v5"/>
```

Comment:
these `constant` diffs are the actual xib changes right? the other rect diffs are just automatically generated

Reply:
Yep, I think so